problems/4.arrays.js

```diff
@@ -1,31 +1,58 @@
 /** Averages the first and last value of an array.  Return null if the array has fewer than 2 elements.
-* @param {number[]} arr - The input array
-* @returns {number} - The average of the first and last values
-*/
-function firstAndLastAverage() {
+ * @param {number[]} arr - The input array
+ * @returns {number} - The average of the first and last values
+ */
+function firstAndLastAverage(arr) {
+    let newArray = [];
+    let first = [i];
+    let last = [arr.lenght - 1]
+    let input1 = Math.floor(first / 2)
+    let input2 = Math.floor(last / 2)
+    for (let i = 0; i < arr.lenght; i++) {
+        const num = arr[i];
+        if (neWarray = inputs1.includes(num) && inputs2.includes(num)); {
+            input.push()
+        }
+        return newArray;
+    }
+
+
 
 }
 
 /** Duplicates all values inside an array.
-* Sample input: [1,4,2,5]
-* Sample output: [1,4,2,5,1,4,2,5]
-* @param {number[]} arr - The input array
-* @returns {number[]} - The duplicated array
-*/
-function duplicateArray() {
-
+ * Sample input: [1,4,2,5]
+ * Sample output: [1,4,2,5,1,4,2,5]
+ * @param {number[]} arr - The input array
+ * @returns {number[]} - The duplicated array
+ */
+function duplicateArray(arr1, arr2) {
```

Comment:
close but duplicate array only takes in one array. 
If you loop through that one array twice your solution would work. 
Also check out concat on MDN.

Reply:
ok. Thank you